### PR TITLE
Linting 2: Electric Boogaloo

### PR DESCRIPTION
**Pull request by superloach**
had git difficulties with the first branch, this one should be good

<details>
<summary>passes with my local <code>golangci-lint</code> config:</summary>

```yml
run:
  concurrency: 4

linters:
  enable-all: true
  disable:
    - gochecknoglobals
    - gochecknoinits
    - gomnd
    - goconst
    - funlen
    - godox
    - gocognit
    - gocyclo
    - gci
  fast: false

issues:
  exclude-rules:
    - path: _test\.go
      linters:
        - deadcode
        - dogsled
        - unparam
        - unused

  include:
    - EXC0002 # disable excluding of issues about comments from golint
```
</details>

we should probably decide on a config to use here, but it's not urgent

### CODE REVIEW

File: assetutil/assetutil.go [hunk 1]
```diff
@@ -39,10 +41,10 @@ func parseConfigFiles(dir string) (confs []config, err error) {
 		},
 	)
 
-	return
+	return confs, err
```
Comment (zacharyburkett) [inline assetutil/assetutil.go:44]: Does the linter really complain about naked returns? I am a fan of them :slightly_smiling_face:

Comment (superloach) [reply assetutil/assetutil.go:44]: I'm not a fan of them, but if you want, I can change that back. `nakedret` is the relevant linter.

File: internal/ebiten/renderer.go [hunk 2]
```diff
@@ -11,14 +11,17 @@ import (
 
 // Renderer is a simple ebiten renderer.
 type Renderer struct {
-	imgs   []engine.Image
+	// unused
+	// imgs   []engine.Image
```
Comment (zacharyburkett) [inline internal/ebiten/renderer.go:15]: We can just remove this

**Review by zacharyburkett — CHANGES_REQUESTED** [review]
Great cleanup of code and docs! This will be a great addition.

Just checkout the two comments I had, and let me know what you think :slightly_smiling_face:

File: internal/ebiten/renderer.go (hunk 2 above)
Comment (superloach) [reply internal/ebiten/renderer.go:15]: gotcha, not sure why I left that in.

**Comment by zacharyburkett** [comment]
As a continuation of the PR or as a separate one, would you be interested in setting up a github workflow for ci linting? #18

**Pull request merged**